docs/backend/upgrading/add-on-upgrade.md

```diff
@@ -0,0 +1,37 @@
+---
+myst:
+  html_meta:
+    "description": "Upgrade Plone add-on products"
+    "property=og:description": "Upgrade Plone add-on products"
+    "property=og:title": "Upgrade Plone add-on products"
+    "keywords": "Upgrade, Plone, add-on, products"
+---
+
+# Upgrade add-on products
+
+```{admonition} Description
+The steps to take to migrate your third party products
+```
+
+- Shut down your Plone server instance.
+- If you specified concrete versions of the third-party products in your *buildout.cfg* file (so-called "pinning", and a recommended practice), like *Products.PloneFormGen* = 1.7.17, update these references to point to the new versions.
+
+```{note}
+Without pinning, i.e. specifying only, for example, *Products.PloneFormGen* and no version, buildout will pick the newest version of the products by default.
+```
+
+- Run *bin/buildout*. Wait until all new software is downloaded and installed.
+- Start Plone again - your site may look weird, or even be inaccessible until you have performed the next step
+- Navigate to the Add-on screen (add `/prefs_install_products_form` to your site URL, and upgrade products if you can (products that support both your current and new version of Plone).
+  : - Perform product-specific upgrade procedures (if any).
+    - You will find these in the documentation of each product.
+
+Should the `/prefs_install_products_form` be unreachable, you should try doing the add-on upgrades from the Management Interface.
+Navigate to the quickinstaller in the Management Interface, and reinstall or upgrade products that are shown to be outdated.
```

Comment:
Where is this, "quickinstaller in the Management Interface"? I cannot find it anywhere in the ZMI.

Reply:
@mauritsvanrees would you please take a look at commit ab70d94633b2e64e619f91f4b3c89d31a1baa124, and provide any feedback? Thank you!

Reply:
Yes, that commit looks fine. Thanks.